MacOSX/HowtoBuildOSX.txt

```diff
@@ -18,6 +18,8 @@ required:
 - cmake (version 3.x and above)
 - svn command line tool (for routino)
 - mercurial (hg) command line tool
```

Comment:
mercurial isn't needed anymore.

Reply:
Could it be necessary for non-QMS packages?

Reply:
no